### PR TITLE
Adds github ISSUE TEMPLATES

### PR DESCRIPTION
**Pull request by pierreozoux**
Like the one from server.

Signed-off-by: pierreozoux <pierre@ozoux.net>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+Nextcloud is an open source project backed by Nextcloud GmbH. Most of our volunteers are home users and thus primarily care about issues that affect home users. Our paid engineers prioritize issues of our customers. If you are neither a home user nor a customer, consider paying somebody to fix your issue, do it yourself or become a customer.
+
+Guidelines for submitting issues:
+
+* Please search the existing issues first, it's likely that your issue was already reported or even fixed.
+    - Go to https://github.com/nextcloud and type any word in the top search/command bar. You probably see something like "We couldn’t find any repositories matching ..." then click "Issues" in the left navigation.
+    - You can also filter by appending e. g. "state:open" to the search string.
+    - More info on search syntax within github: https://help.github.com/articles/searching-issues
+    
+* This repository https://github.com/nextcloud/server/issues is *only* for issues within the Nextcloud Server code. This also includes the apps: files, encryption, external storage, sharing, deleted files, versions, LDAP, and WebDAV Auth
```
Comment (fschrempf) [inline .github/ISSUE_TEMPLATE/Bug_report.md:21]: We need to change this for the groupfolders repository

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+**Operating system:**
+
+**Web server:**
+
+**Database:**
+
+**PHP version:**
+
+**Nextcloud version:** (see Nextcloud admin page)
```
Comment (fschrempf) [inline .github/ISSUE_TEMPLATE/Bug_report.md:60]: I guess we should also add the groupfolders app version

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 3]
```diff
@@ -0,0 +1,1219 @@
+
```
Comment (fschrempf) [inline .github/ISSUE_TEMPLATE/Feature_request.md:1]: This file doesn't look right at all. I guess there should be Markdown and no HTML here.

**Comment by fschrempf** [comment]
Having issue templates is definitely a good thing, but the ones from the server seem a bit too long and confusing to me. And also we can't use them as-is, as there are references to the server repository and other things that would need changes. See my inline comments.

**Comment by pierreozoux** [comment]
Ok, was a bit fast, Ill review next weekand improve thanks for feedback.

**Comment by pierreozoux** [comment]
Ok, sorry :) 

I think I fixed the PR, ready to be reviewd one more time :)

**Comment by fschrempf** [comment]
> Ok, sorry :)
> 
> I think I fixed the PR, ready to be reviewd one more time :)

No worries! Thanks!

Can you squash all the commits into one and force-push? I think we are ready to merge then and this should also fix the DCO check issue as your first commit is signed-off correctly.

**Comment by pierreozoux** [comment]
In GitLab you can merge/squash.. it is always a mess to rebase/force push.. anyway :) It is done :)

**Review by icewind1991 — APPROVED** [review]

**Review by fschrempf — APPROVED** [review]

**Pull request merged**